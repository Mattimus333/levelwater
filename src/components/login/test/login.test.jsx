import React from 'react';
import { shallow } from 'enzyme';
import Login from '../login.jsx';
import toJson from 'enzyme-to-json';


describe('login', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<Login />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
