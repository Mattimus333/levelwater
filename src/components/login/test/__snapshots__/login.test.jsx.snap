exports[`login renders a snapshot 1`] = `
<div
  id="login">
  <Connect(ReduxForm) />
  <OAuthButton />
</div>
`;
